packages/fe-mockserver-core/src/middleware.ts

```diff
@@ -126,3 +126,26 @@ export async function createMockMiddleware(
 
     await Promise.all(oDataHandlerPromises);
 }
+
+/**
+ * Exracts JSON response from multipart content in case of changeset batch
+ *
+ * @param batchResponse
+ */
+
+export function getJsonFromMultipartContent(batchResponse: string) {
```

Comment:
This seems to be only test code, can you please ensure it's only in test sources ?

Reply:
done